src/app/ApplicationTemplate.Client/ApplicationTemplate.Client.csproj

```diff
@@ -17,7 +17,7 @@
 		<PackageReference Include="Microsoft.Extensions.Http" Version="3.1.0" />
 		<PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="3.1.0" />
 		<PackageReference Include="Reactive.Annex" Version="0.6.0-dev.50" />
-		<PackageReference Include="Refit" Version="4.7.51" />
+		<PackageReference Include="Refit" Version="6.3.2" />
```

Comment:
Something to keep in mind, we've recently updated refit to 6.3.2 and they did a breaking changes with the way they're handling enums https://github.com/reactiveui/refit/issues/1103. Depending on projects, it can be problematic